indico/modules/events/contributions/templates/management/exportdialog.html

```diff
@@ -0,0 +1,11 @@
+{% from 'forms/_form.html' import simple_form  %}
+{% from 'message_box.html' import message_box %}
+
+{% call message_box('highlight', large_icon=true) %}
+    {% trans -%}
+    Please select the  desired format and sort order.
```

Comment:
indentation, double space

Reply:
Maybe adding this message is overkill. If you don't use it you can even get rid of this template altogether and just use `jsonify_form()` in the Python code.

If you do keep the template, rename it to `export_dialog.html` please